bus/meson.build

```diff
@@ -43,6 +43,27 @@ else
                              required: true).path()
 endif
 
+launcher_args = [
+             '-DSYSCONFDIR="@0@"'.format(atspi_sysconfdir),
+             '-DDATADIR="@0@"'.format(atspi_datadir),
+             '-DDBUS_DAEMON="@0@"'.format(dbus_daemon),
+           ]
+
+if get_option('dbus_broker') != 'default'
+  launcher_args += '-DDBUS_BROKER="@0@"'.format(get_option('dbus_broker'))
+else
+  dbus_broker = find_program('dbus-broker-launch',
+                             '/sbin/dbus-broker-launch',
+                             '/usr/sbin/dbus-broker-launch',
+                             '/libexec/dbus-broker-launch',
+                             '/usr/libexec/dbus-broker-launch',
+                             '/usr/pkg/bin/dbus-broker-launch',
+                             required: false)
```

Comment:
Is there any reason why `dbus-broker-launch` would not be in path? All those paths seem rather unnecessary.

Reply:
I followed the logic used for `dbus-daemon` for the sake of consistency.

Reply:
dbus-broker-launch is not going to be installed in any of those paths, so I don't think consistency is important at all here.

Reply:
I don't have a strong opinion on this. I expect also dbus-daemon could rely on the binary being in the path. I'll wait for the maintainer to comment and go with whatever they prefer.

Reply:
Cool; thanks for the patch. I apologize for not looking at it sooner.
I would say that it's a good idea to make the default configurable, since a couple of people are asking for it, and I don't think that it would do any harm.

Reply:
I pushed  a new version now, addressing all the feedback.